azurerm/resource_arm_iothub.go

```diff
@@ -476,6 +504,12 @@ func resourceArmIotHubRead(d *schema.ResourceData, meta interface{}) error {
 		if err := d.Set("fallback_route", fallbackRoute); err != nil {
 			return fmt.Errorf("Error setting `fallbackRoute` in IoTHub %q: %+v", name, err)
 		}
+
+		ipFilterRules := flattenIPFilterRules((properties.IPFilterRules))
```

Comment:
we can remove the duplicate `()` here 😄

```suggestion
		ipFilterRules := flattenIPFilterRules(properties.IPFilterRules)
```